tests/api/test_dnf_package.py

```diff
@@ -0,0 +1,255 @@
+
+    def test_prereq_ignoreinst(self):
+        # Package.prereq_ignoreinst
+        self.assertHasAttr(self.package, "prereq_ignoreinst")
+        self.assertHasType(self.package.prereq_ignoreinst, list)
+
+    def test_rpmdbid(self):
+        # Package.rpmdbid
+        self.assertHasAttr(self.package, "rpmdbid")
+        self.assertHasType(self.package.rpmdbid, int)
+
+    def test_source_debug_name(self):
+        # Package.source_debug_name
+        self.assertHasAttr(self.package, "source_debug_name")
+        self.assertHasType(self.package.source_debug_name, str)
+
+    def test_source_name(self):
+        # Package.source_name
+        self.assertHasAttr(self.package, "source_name")
+        self.assertHasType(self.package.source_name, str)
+
+    def test_debugsource_name(self):
+        # Package.debugsource_name
+        self.assertHasAttr(self.package, "debugsource_name")
+        self.assertHasType(self.package.debugsource_name, str)
+
+    def test_sourcerpm(self):
+        # Package.sourcerpm
+        self.assertHasAttr(self.package, "sourcerpm")
+        self.assertHasType(self.package.sourcerpm, str)
+
+    def test_suggests(self):
+        # Package.suggests
+        self.assertHasAttr(self.package, "suggests")
+        self.assertHasType(self.package.suggests, list)
+
+    def test_summary(self):
+        # Package.summary
+        self.assertHasAttr(self.package, "summary")
+        self.assertHasType(self.package.summary, str)
+
+    def test_supplements(self):
+        # Package.supplements
+        self.assertHasAttr(self.package, "supplements")
+        self.assertHasType(self.package.supplements, list)
+
+    def test_url(self):
+        # Package.url
+        self.assertHasAttr(self.package, "url")
+        # The value is missing here so the type is type(None) but it should be str
+        # self.assertHasType(self.package.url, str)
+        self.package.url
+
+    def test_version(self):
+        # Package.version
+        self.assertHasAttr(self.package, "version")
+        self.assertHasType(self.package.version, str)
+
+    def test_packager(self):
+        # Package.packager
+        self.assertHasAttr(self.package, "packager")
+        # The value is missing here so the type is type(None) but it should be str
+        # self.assertHasType(self.package.packager, str)
+        self.package.packager
+
+    def test_remote_location(self):
+        # Package.remote_location
+        self.assertHasAttr(self.package, "remote_location")
+        # This fails due to a bug (filed RhBug:1873146)
+        #self.package.remote_location(schemes='http')
+
+    def test_debuginfo_suffix(self):
+        # Package.DEBUGINFO_SUFFIX
+        self.assertHasAttr(self.package, "DEBUGINFO_SUFFIX")
+        self.package.DEBUGINFO_SUFFIX
+
+    def test_debugsource_suffix(self):
+        # Package.DEBUGSOURCE_SUFFIX
+        self.assertHasAttr(self.package, "DEBUGSOURCE_SUFFIX")
+        self.package.DEBUGSOURCE_SUFFIX
```

Comment:
These two variables can be also tested for type: ``DEBUGINFO_SUFFIX`` and ``DEBUGSOURCE_SUFFIX``

Reply:
Added, thanks!